docs/integrations/amazon-aws/amazon-elastic-container-service-container-insights-cloudwatch.md

```diff
@@ -0,0 +1,643 @@
+---
+id: elastic-container-service-container-insights-cloudwatch
+title: Amazon Elastic Container Service (ECS) using Container Insights and CloudWatch
+sidebar_label: Amazon ECS with Container Insights and CloudWatch
+description: Provides preconfigured searches and Dashboards that allow you to monitor various metrics.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('img/integrations/amazon-aws/ecs.png')} alt="Thumbnail icon" width="50"/>
+
+Amazon Elastic Container Service (Amazon ECS) is a container management service that allows you to manage Docker containers on a cluster of Amazon EC2 instances. The Sumo Logic App for Amazon ECS provides preconfigured searches and Dashboards that allow you to monitor various metrics (CPU and Memory Utilization, CPU and Memory Reservation) across ECS clusters and services. The App also monitors API calls made by or on behalf of Amazon ECS in your AWS account.
+
+We have two app versions which have separate collection steps 
```

Comment:
add colon after

Reply:
Added

Reply:
I'm not seeing the colon above. Where are you making these changes?